elasticsearch-2.4/Dockerfile

```diff
@@ -0,0 +1,146 @@
+###
+# Install OpenJDK JRE 8
+# https://raw.githubusercontent.com/docker-library/openjdk/master/8-jre/Dockerfile
+#
+# NOTE: THIS DOCKERFILE IS GENERATED VIA "update.sh"
+#
+# PLEASE DO NOT EDIT IT DIRECTLY.
+#
+
+FROM localhost:5000/ubuntu
```

Comment:
Whoa! I have never seen this notation. Does this mean a local image?

Reply:
Yes, Tugboat runs it's own Docker registry, and because of how Tugboat takes mounts the filesystem, we can't use the official image for elasticsearch. So this PR is a concatenation and adaptation of the official openjdk and elasticsearch images.